test-clients/src/main/java/com/hedera/services/bdd/spec/queries/QueryVerbs.java

```diff
@@ -108,9 +110,18 @@ public static HapiGetContractRecords getContractRecords(final String contract) {
 
 	public static HapiContractCallLocal contractCallLocal(
 			final String contract,
-			final String abi,
-			final Object... params
-	) {
+			final String functionName,
+			final Object... params) {
+		final var abi = getABIFor(FUNCTION, functionName, contract);
+		return new HapiContractCallLocal(abi, contract, params);
+	}
+
+	/** This function provides for the proper execution of specs, which execute contract local calls with a function ABI
+	 instead of function name
+	 */
+	public static HapiContractCallLocal contractCallLocalWithFunctionAbi(final String contract,
```

Comment:
Same as with TxnVerbs.contractCallWithFunctionAbi(), we should remove this method and call the contractCallLocal with function name. If the function is missing from the ABI file, add it manually there.

Reply:
Similarly for the previous comment, related to the contractCallLocalWithFunctionAbi() - after a discussion with the reviewer was decided the method not be removed.